zuul.d/project-templates.yaml

```diff
@@ -330,6 +330,9 @@
         # NOTE(icey) BUT REALLY, DO NOT ENABLE THE FOLLOWING UNTIL YOU KNOW WE CAN
         # RUN A 3.10 JOB ON ZOSCI.
         #- tox-py310
+    promote:
+      jobs:
+        - charmhub-osci-sync
```

Comment:
something I'm not sure if this is the right place to trigger the job, I don't know if there should be a new check job that each charm needs/should opt-in.

Reply:
@ChrisMacNaughton what do you think?